src/inverse.jl

```diff
@@ -161,4 +161,7 @@ inverse(f::Base.Fix2{typeof(invdivrem)}) = Base.Fix2(divrem, f.x)
 inverse(f::Base.Fix2{typeof(fldmod)}) = Base.Fix2(invfldmod, f.x)
 inverse(f::Base.Fix2{typeof(invfldmod)}) = Base.Fix2(fldmod, f.x)
 
+inverse(::typeof(reim)) = Base.splat(complex)
```

Comment:
Is Base.splat official API? I think generally packages (including this one) should not use Julia internals.

Reply:
Ahh, good catch - `Base.splat` is official, but only since Julia v1.8 or v1.9, I think?

Reply:
So we'll need a `static if VERSION ...` there, with a different implementation for older Julia versions.

Reply:
Ah, sorry, I have to correct myself - it is documented since Julia v1.0!

Reply:
I remembered wrong - it's `Base.Splat` that is new, `Base.splat` we've had since 1.0.